content/posts/newsletter-012/index.md

```diff
@@ -62,6 +62,59 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [ochre][4k-post] - 4K Intro
+
+[![Youtube preview: mountains & spheres](4k-into-youtube.jpeg)][4k-video]
+
+^ _Click to [watch the demo on Youtube][4k-video]._
+
+Jani Peltonen has recently released a [4K intro][4k-src]
+which is completely written in Rust and GLSL
+and published an article ["Writing a winning 4K intro in Rust"][4k-post]:
+
+> A 4K intro is a demo where the entire program (including any data)
+> has two be 4096 bytes or less so it is important
```

Comment:
Presumably this should be 'has to be', but that's a mistake from the original post, so might be fine to leave it in.